Solutions/FlareSystemsFirework/Data/Solution_FlareSystemsFirework.json

```diff
@@ -1,15 +1,20 @@
 {
   "Name": "FlareSystemsFirework",
-  "Author": "Nikhil Tripathi - v-ntripathi@microsoft.com",
+  "Author": "Microsoft - support@microsoft.com",
```

Comment:
Can you get confirmation on the Author... as it's an ISV solution, we supposed to add partner details